cuda/test/solver/lower_trs_kernels.cpp

```diff
@@ -174,21 +174,31 @@ TEST_F(LowerTrs, CudaMultipleRhsApplySyncfreeIsEquivalentToRef)
 TEST_F(LowerTrs, CudaMultipleRhsApplyIsEquivalentToRef)
 {
     initialize_data(50, 3);
+#if CUDA_VERSION >= 11031
+#if CUDA_VERSION < 12000
+    // The cuSPARSE Generic SpSM implementation uses the wrong stride here
+    // so the input and output stride need to match
+    auto in_stride = 4;
+    auto out_stride = 4;
+#else
+    // The cuSPARSE 12 solver is even worse: It only works if the stride is
+    // equal to the number of columns.
+    auto in_stride = 3;
+    auto out_stride = 3;
```

Comment:
Is this with 12 only? I thought I also had to pick stride = nrhs for 11.7, but maybe I misremember. It's puzzling this decrease in stride support over time.

Reply:
Might be that they introduced the new algorithm at 11.7 already, I only remember hearing there were some changes around 12.